unit-tests/unit-test-config.py

```diff
@@ -39,28 +39,29 @@ def usage():
 if not os.path.isdir( dir ) or not os.path.isdir( builddir ):
     usage()
 
-# Parse command-line:
-try:
-    opts, args = getopt.getopt( sys.argv[3].split(), 'hr:t:',
-                                longopts=['help', 'regex=', 'tag=', 'list-tags', 'list-tests'] )
-except getopt.GetoptError as err:
-    log.e( err )  # something like "option -a not recognized"
-    usage()
 regex = None
 required_tags = []
 list_tags = False
 list_tests = False
-for opt, arg in opts:
-    if opt in ('-h', '--help'):
+# if we received flags, parse command-line:
+if len( sys.argv ) > 3:
```

Comment:
3? Why is the if needed?

Reply:
![image](https://user-images.githubusercontent.com/70845604/117938846-236b1100-b310-11eb-9900-a7814c5892b6.png)
options is the 4th argument. If there are no options I can't do the `split()` and get an error

Reply:
Wait... I just noticed the `sys.argv[3].split()`... you're saying all the options end up as one argument?
I think we need to have the python work from anywhere, and not just from the CMake command -- i.e., we need to fix this.